pkg/vm/engine/disttae/database.go

```diff
@@ -95,6 +95,31 @@ func (db *database) Delete(ctx context.Context, name string) error {
 	return nil
 }
 
+func (db *database) Truncate(ctx context.Context, name string) error {
```

Comment:
If you are going to put the Truncate in the database instead of the table, remove the table.Truncate

Reply:
ok,no problem!